letsencrypt/client/standalone_authenticator.py

```diff
@@ -266,30 +266,23 @@ def already_listening(self, port):  # pylint: disable=no-self-use
         If so, also tell the user via a display notification.
 
         .. warning::
-            The current implementation is Linux-specific.  (On other
-            operating systems, it will simply not detect bound ports.)
-            This function can only usefully be run as root.
+            On some operating systems, this function can only usefully be
+            run as root.
 
         :param int port: The TCP port in question.
         :returns: True or False."""
 
+        listeners = [conn.pid for conn in psutil.net_connections() \
+                     if conn.status == 'LISTEN' and \
+                     conn.type == socket.SOCK_STREAM and \
+                     (conn.laddr == ('0.0.0.0', port) or \
```

Comment:
I thought that this behavior would help when something has only bound the port on the loopback interface and not on a public interface. But I checked and that's not the case (it's still not possible for us to bind the port in that circumstance), so I filed #255 to think about what the right behavior is.

I will change this to `conn.laddr[1] == port`.